dashboard/app/controllers/datasets_controller.rb

```diff
@@ -6,7 +6,8 @@ class DatasetsController < ApplicationController
   before_action :initialize_firebase
   authorize_resource class: false
 
-  LIVE_DATASETS = ['Daily Weather', 'Top 200 USA', 'Top 200 Worldwide', 'Viral 50 USA', 'Viral 50 Worldwide']
+  LIVE_DATASETS = ['Daily Weather', 'Top 200 USA', 'Top 200 Worldwide', 'Viral 50 USA', 'Viral 50 Worldwide',
+                   'COVID-19 Cases per US State', 'COVID-19 Cases per Country']
```

Comment:
NIT: Should these all be on their own line? (sorry for the style comment... don't know what our linter's rules are for ruby)

Reply:
This passes the linter, so I think it's fine